brayns/common/scene/Scene.cpp

```diff
@@ -517,9 +519,46 @@ VolumeHandlerPtr Scene::getVolumeHandler()
     {
         const std::string& volumeFile =
             _parametersManager.getVolumeParameters().getFilename();
-        if( !_volumeHandler && !volumeFile.empty() )
-            _volumeHandler.reset( new VolumeHandler(
-                _parametersManager.getVolumeParameters(),  volumeFile ));
+        const std::string& volumeFolder =
+            _parametersManager.getVolumeParameters().getFolder();
+
+        if( !_volumeHandler && ( !volumeFile.empty() || !volumeFolder.empty() ))
+        {
+            _volumeHandler.reset(
+                new VolumeHandler( _parametersManager.getVolumeParameters(), TM_MODULO ));
```

Comment:
Much better indeed. But now I would need to change it everywhere else to be coherent. Done for this one, I will do the rest in a separate PR.